metrics/web/test/common/presentation/widgets/in_progress_animated_bar_test.dart

```diff
@@ -0,0 +1,200 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:metrics/base/presentation/widgets/rive_animation.dart';
+import 'package:metrics/common/presentation/metrics_theme/config/dimensions_config.dart';
+import 'package:metrics/common/presentation/widgets/in_progress_animated_bar.dart';
+import 'package:rive/rive.dart';
+
+// ignore_for_file: avoid_redundant_argument_values
+
+void main() {
+  group("InProgressAnimatedBar", () {
+    const asset = 'web/animation/in_progress_bar.riv';
+    const artboardName = 'In progress bar';
+
+    final controller = SimpleAnimation('Animation 1');
+
+    final riveAnimationFinder = find.byType(RiveAnimation);
+
+    RiveAnimation getRiveAnimation(WidgetTester tester) {
+      return tester.widget<RiveAnimation>(riveAnimationFinder);
+    }
+
+    testWidgets(
+      "throws an AssertionError if the given height is null",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _InProgressAnimatedBarTestbed(height: null),
+        );
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "throws an AssertionError if the given rive asset is null",
+      (WidgetTester tester) async {
+        await tester.pumpWidget(
+          const _InProgressAnimatedBarTestbed(riveAsset: null),
+        );
+
+        expect(tester.takeException(), isAssertionError);
+      },
+    );
+
+    testWidgets(
+      "applies the given height",
```

Comment:
```suggestion
      "applies the given height to the animated bar",
```
Please, apply this suggestion to all tests below.